src/store/mutations.ts

```diff
@@ -53,6 +53,60 @@ const mutations: MutationTree<CalChartState> = {
     const currentSS = getSelectedStuntSheet(state);
     currentSS.addDot(dot);
   },
+  syncDotLabelIndices(
+    state,
+    payload: { currentSSDotIndex: number; nextSSDotIndex: number }
+  ): void {
+    // TODO: Design a better way to connect dots between stuntsheets
```

Comment:
What if there was a sort of linked list linking dots to one another? Like in `StuntSheetDot.ts` there was a `nextDotIndex` that is just the index of the dot that's associated with it in the next stuntsheet. I think it would be easier to assign the next dot that way, and propogating the dot labels would have to be done through the list.